src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out

```diff
@@ -5,7 +5,7 @@ create_reference_table
 
                
 step s2-load-metadata-cache: 
-	COPY test_reference_table FROM PROGRAM 'echo "1\n2\n3\n4\n5"';
+	COPY test_reference_table FROM PROGRAM 'echo 1 && echo 2 && echo 3 && echo 4 && echo 5';
```

Comment:
These files also have similar echos which have been missed from the current PR:
- isolation_create_distributed_table
- isolation_master_append_table
- isolation_master_apply_delete

Reply:
What do you think about changing the following files?

    isolation_create_distributed_table
    isolation_master_append_table
    isolation_master_apply_delete

Reply:
Ok, I did this. Please take a look.